sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -904,8 +904,11 @@ public void promptForDockerizedDb(Server server, DockerHelper dockerHelper, Stri
 			throws MojoExecutionException {
 		promptForDockerHostIfMissing(dockerHelper, dockerHost);
 
-		String containerId = prompt(
-				"Please specify your container id/name/label (you can get it using command `docker ps -a --no-trunc`)");
+		List<String> containerNames = dockerHelper.getDockerContainerNames();
+		String customMessage = "Please specify your container id/name/label (you can get it using command `docker ps -a --no-trunc`) ";
```

Comment:
Are you combining selecting with specify a container id/name/label or replacing the latter with the former?

Reply:
If users select the **other** option, they are prompted with: 'Please specify your container ID...'"